lib/widgets/category_item.dart

```diff
@@ -16,7 +16,6 @@ class CategoryItem extends StatelessWidget {
   );
 
   void selectCategory(BuildContext ctx) {
-
     Navigator.of(ctx).pushNamed(
       CategoryMealsScreen.routeName,
```

Comment:
it would be better if this value is part of this widget's constructor parameters.